packages/common/src/fixtures/xforms.ts

```diff
@@ -115,13 +115,23 @@ export class XFormResource<Type extends XFormResourceType> {
 	): XFormResource<'remote'> {
 		const resourceURL = new URL(remoteURL);
 		const loadXML = xformURLLoader(resourceURL);
+		const attachmentService = () => {
```

Comment:
This loads all images from the preview form: `All Question Types`. Because this form is loaded with a URL parameter: `http://localhost:5173/#/form?url=http://localhost:5173/@fs/Users/x/Documents/web-forms/packages/common/src/fixtures/preview-service/xforms/all-question-types-v2024091201.xml`

It works in the local environment, but I wonder if it does it too on the website